lib/helpers.js

```diff
@@ -192,14 +195,14 @@ const getSolidityVersion = fileContents => {
 
             if (solidityVersion.indexOf('<') !== -1) {
                 if (solidityVersion.indexOf('<=') !== -1) {
-                    solidityVersion = solidityVersion.substring(solidityVersion.length-5, solidityVersion.length);
+                    solidityVersion = solidityVersion.substring(solidityVersion.length - 5, solidityVersion.length);
                 } else {
-                    upperLimit = solidityVersion.substring(solidityVersion.length-5, solidityVersion.length);
+                    upperLimit = solidityVersion.substring(solidityVersion.length - 5, solidityVersion.length);
                 }
-            }else if (solidityVersion.indexOf('>') !== -1) {
+            } else if (solidityVersion.indexOf('>') !== -1) {
                 solidityVersion = releases.latest;
             } else {
-                upperLimit = '0.' + (parseInt(solidityVersion[2])+1).toString() + '.0'
+                upperLimit = '0.' + (parseInt(solidityVersion[2]) + 1).toString() + '.0';
             }
```

Comment:
@eswarasai Thanks for your correcting my spelling mistakes. I think I should extend a plugin for js in my vs code.

Reply:
Sure. No problem.

Yeah, about linting, it's always better to have good IDE support which will ease up development. Also, I'll be setting up precommit hooks to fix the lint issues before committing the code for the ease of maintenance and code review :)